sample.env

```diff
@@ -0,0 +1,6 @@
+DATABASE_URL=postgresql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME}
+DATABASE_NAME=test
+DATABASE_PASSWORD=test
+DATABASE_USER=test
+DATABASE_PORT=5432
+DATABASE_HOST=localhost
```

Comment:
This looks more like a `test.env` rather than `sample.env`. I'd create an `example.env` with just keys without any values